app/Http/Controllers/UserApiController.php

```diff
@@ -62,9 +63,21 @@ public function getAchievements()
     /**
      * @return Collection|OrderLine[]
      */
-    public function getPurchases()
+    public function getPurchases(Request $request)
```

Comment:
Does this not break any other uses of this method?

Reply:
Are there other usages?

Reply:
Yeah, not sure, just checking.